### PR TITLE
fix logo

### PR DESCRIPTION
**Pull request by srilq**

### CODE REVIEW

**Comment by srilq** [comment]
I called it `260`, but I actually meant `460`... I'll change this now.

**Comment by srilq** [comment]
@almostagile 
screen shot 2017-11-28 at 11 28 05

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <p align="center">
-  <img src="https://raw.githubusercontent.com/conde-nast-international/flyway-api-docs/master/source/images/logo.png" width="270">
+  <img src="https://raw.githubusercontent.com/conde-nast-international/flyway-api-docs/master/source/images/logo_black.png" width="325">
 </p>
 
 <p align="center">Generate the documentaion for Condé Nast International Flyway Integration API using Slate.</p>
 
+<p align="center"><em>This repository is a fork of Slate. Check it out at <a href="https://lord.github.io/slate">lord.github.io/slate</a>.</em></p>
```
Comment (almostagile) [inline README.md]: Actually that's the wrong link. That's the demo of Slate, best to link to the GitHub repo.

File: source/layouts/layout.erb [hunk 2]
```diff
@@ -51,7 +51,7 @@ under the License.
       </span>
     </a>
     <div class="toc-wrapper">
-      <%= image_tag "logo.png", class: 'logo' %>
+      <%= image_tag "logo_white_460.png", class: 'logo' %>
```
Comment (almostagile) [inline source/layouts/layout.erb]: Changing this means we are diverging from Slate docs https://github.com/lord/slate/wiki/Changing-the-Logo. Not very important but I'd prefer to follow the convention. Still not clear why we need multiple sizes. Seems like we could just use the larger ones.

**Pull request merged**